src/components/DeckPreview.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import AgeText from './AgeText';
+
+class DeckPreview extends React.Component {
```

Comment:
Same comment as above. This can (and should) be written as a functional component.